codec.go

```diff
@@ -25,7 +30,10 @@ type framedMsgpackEncoder struct {
 }
 
 func newFramedMsgpackEncoder(writeCh chan []byte, resultCh chan error) *framedMsgpackEncoder {
-	mh := &codec.MsgpackHandle{WriteExt: true}
+	mh := &codec.MsgpackHandle{
```

Comment:
I think this should be pulled out into its own helper function, like `newMsgpackHandle()` or something, just to make sure it's used everywhere.